LandMark/App/AppCoordinator.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  AppCoordinator.swift
+//  LandMark
+//
+//  Created by Lucas Migge on 30/10/24.
+//
+
+import UIKit
+
+protocol Coordinator {
+    var children: [Coordinator] { get set }
+    var navigationController: UINavigationController { get set }
+    func start()
```

Comment:
Pq ambos `get` e `set` públicos? Idealmente não vamos ter ninguém de fora setando nada em um Coordinator além dele mesmo, certo? Então bastaria declarar essas propriedades aqui como `{ get }`?

```suggestion
    var children: [Coordinator] { get }
    var navigationController: UINavigationController { get }
    func start()
```

Eu argumentaria que nem o `get` deveria ser público, dado que tudo deveria ser feito através de outros métodos, mas aí entra em toda a questão de se valeria ter um protocolo para isso ou não, que não acho que vale a gente entrar dado o contexto e tempo desse projeto.

Reply:
Justo!